k8s/production/gitlab/release.yaml

```diff
@@ -84,9 +88,11 @@ spec:
         database: gitlabhq_production
         username: gitlab
 
+      # This block will be overwritten by the TF-managed configmap
+      # specified in the `valuesFrom` block above.
       redis:
-        password:
-          enabled: true
+        auth:
+          enabled: false
```

Comment:
One more question, why does this need to be specified here? Isn't it specified in the config map as well?

Reply:
Yeah, good catch, i missed that when I removed the hostname. We don't need this block at all.